.cico/cico_openshift_e2e.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+set -ex
```

Comment:
Seems `.cico/cico_openshift_e2e.sh` should be executable
![Screenshot_20200609_101905](https://user-images.githubusercontent.com/5887312/84117771-b5043b80-aa3a-11ea-8595-0e635cb309ae.png)